mgmtfn/k8splugin/README.md

```diff
@@ -0,0 +1,311 @@
+# Contiv Networking for Kubernetes
+
+Contiv is integrated with Kubernetes via a CNI plugin. This integration enables you
+to set up and manage inter-pod connectivity in a Kubernetes cluster using Contiv 
+policy model.
+
+## Getting Started
+
```

Comment:
Making/building binaries will go away soon -- there is a follow up PR coming.
Checking out repos -- the ansible repo will need to be checked out, unless we want to provide a .tgz.

I did not quite follow this one:
"ansible work to install things (those all should be part of vbox image, for Vagrant)"

The ansible repo is there and the packages are all pre-installed in the vbox.

In terms of the steps, are you suggesting there should be 4 steps instead of the 7 that I have?